frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -99,24 +97,28 @@
               <hr class="mt-6" />
               <h4 id="how-it-was-made" class="is-info is-size-4">How it was made</h4>
               <p>
-                Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a
-                piece of classical Latin literature from 45 BC, making it over 2000 years old.
-                Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked
-                up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and
-                going through the cites of the word in classical literature, discovered the
-                undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de
-                Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45
-                BC. This book is a treatise on the theory of ethics, very popular during the
-                Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes
-                from a line in section 1.10.32.
+                Data in this database are predicted from available advanced deep learning tools:
```

Comment:
This sounds like the method is called k<sub>cat</sub> but it's not. What about we add a citation like format with link to the publication?
e.g. 
                Data in this database are predicted from available advanced deep learning tools ([Li et al. Nature Catalysis](https://doi.org/10.1038/s41929-022-00798-z))

Reply:
To people in the field, k<sub>cat</sub> is a known term. It's also easily googleable. I'd rather suggest something along the lines of `prediction of k<sub>cat</sub> with _DLKcat_ published in ...`.

Reply:
That sounds much better.